Cargo.toml

```diff
@@ -55,5 +56,6 @@ pcsaft = ["association"]
 gc_pcsaft = ["association"]
 uvtheory = ["lazy_static"]
 pets = []
-python = ["pyo3", "numpy", "feos-core/python", "feos-dft?/python"]
+rayon = ["rayon_", "ndarray/rayon", "feos-core/rayon"]
```

Comment:
maybe `rayon = ["dep:rayon", ...]` then the `package` should become obsolete

Reply:
Added `"dep:rayon"` and renamed the import name in the dependency list and accordingly within the code where it is needed.